propolis/Cargo.toml

```diff
@@ -8,7 +8,8 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-libc = "0.2"
+#libc = "0.2"
+libc = { git = "https://github.com/rust-lang/libc.git" }
```

Comment:
Waiting on getting a new release on crates.io (https://github.com/rust-lang/libc/pull/2383) since I needed to add the illumos bindings for preadv/pwritev (https://github.com/rust-lang/libc/pull/2382).

Reply:
How about specifying the rev required, and include a short comment mentioning the reason

Reply:
That works too, just hoped for getting a new release out quicker as the initial PR was relatively quick. But git rev works too